source/cloud-security/gcp/prerequisites/considerations.rst

```diff
@@ -84,30 +84,3 @@ Below there is an example of a configuration that uses more than one bucket:
 
   </gcp-bucket>
   
-
-Enabling dashboard visualization  
---------------------------------
-  
-You can activate the corresponding Security Information Management module on the Wazuh Dashboard. This module provides additional details and insights about events, as shown in the screenshots below.
-
-    .. thumbnail:: /images/gcp/gcp-dashboard.png
-       :title: GCP dashboard
-       :alt: GCP dashboard
-       :align: center
-       :width: 80%
-
-    .. thumbnail:: /images/gcp/gcp-events.png
-       :title: GCP events
-       :alt: GCP events
-       :align: center
-       :width: 80%
```

Comment:
**thought**: this could require rephrasing because the visibility of the module is not configurable.

Reply:
Ok, I will remove the related images too.